src/api/spec/factories/staged_request.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :staged_request do
+    bs_request { nil }
+    project { nil }
```

Comment:
Factories are supposed to work by default and you validate the presence of project/request.